apps/fluent-tester/src/TestComponents/Menu/MenuTest.tsx

```diff
@@ -152,6 +152,29 @@ const MenuSubMenu: React.FunctionComponent = () => {
   );
 };
 
+const ScrollViewMenuPopover = MenuPopover.customize({ maxHeight: 100 });
```

Comment:
Why not just pass as a prop?

Reply:
Leaving this one as is for the mutable test case. Added another one for custom height where I pass in `maxHeight` as a prop. see cbc482ce01464538b2ca4bd6a83e0a892dc09858